account_invoice_refund_option/tests/test_account_invoice_refund_option.py

```diff
@@ -0,0 +1,31 @@
+from .common import AccountInvoiceRefundOptionTest
```

Comment:
Why a separate file for these tests? :thinking:

Reply:
@ivantodorovich : I keep in separate to define static variables from actual test case